### PR TITLE
fix: remove duplicate builder execution

### PR DESCRIPTION
**Pull request by caalador**
The builder was executed twice, most
likely due to a merge conflict.

### CODE REVIEW

File: flow-tests/test-application-theme/test-theme-switch-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeSwitchLiveReloadIT.java [hunk 1]
```diff
@@ -78,10 +81,10 @@ private void waitUntilThemeSwap(String errMessage,
                                     SerializableSupplier<Boolean> themeStylesSupplier) {
         int attempts = 0;
         while (attempts < ATTEMPTS) {
-            getDriver().navigate().refresh();
+            open();
+            getCommandExecutor().waitForVaadin();
```
Comment (joheriks) [inline flow-tests/test-application-theme/test-theme-switch-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeSwitchLiveReloadIT.java:85]: Isn't `waitForVaadin` already called by superclass method `AbstractTestBenchTest::open`?

File: flow-tests/test-application-theme/test-theme-switch-live-reload/pom.xml [hunk 2]
```diff
@@ -61,9 +61,9 @@
                     <scanIntervalSeconds>1</scanIntervalSeconds>
                     <scanTargetPatterns>
                         <scanTargetPattern>
-                            <directory>${project.basedir}</directory>
+                            <directory>${project.basedir}/src/main/java</directory>
                             <includes>
-                                <include>**/AppShell.java</include>
+                                <include>**/ThemeSwitchLiveReloadView.java</include>
```
Comment (joheriks) [inline flow-tests/test-application-theme/test-theme-switch-live-reload/pom.xml:66]: Shouldn't this also be applied in master? Could we have a separate commit for the test renovation, CP it to relevant branhces, then rebase this PR?

**Review by joheriks — CHANGES_REQUESTED** [review]

File: flow-tests/test-application-theme/test-theme-switch-live-reload/pom.xml (hunk 2 above)
Comment (caalador) [reply flow-tests/test-application-theme/test-theme-switch-live-reload/pom.xml:66]: Master has the AppShell class where as v14 branch doesn't so it's not applicable to master.
The test worked with the wrong class apparently due to the fact that the work was done in advance.

File: flow-tests/test-application-theme/test-theme-switch-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeSwitchLiveReloadIT.java (hunk 1 above)
Comment (caalador) [reply flow-tests/test-application-theme/test-theme-switch-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ThemeSwitchLiveReloadIT.java]: Reverted test class change.

**Review by joheriks — APPROVED** [review]

**Pull request merged**